src/Emogrifier/CssInliner.php

```diff
@@ -626,6 +626,25 @@ private function attributeValueIsImportant($attributeValue)
         return \strtolower(\substr(\trim($attributeValue), -10)) === '!important';
     }
 
+    /**
+     * Gets the array of selectors present in the CSS provided to `inlineCss()` for which the declarations could not be
+     * applied as inline styles, but which may affect elements in the HTML.  The relevant CSS will have been placed in a
+     * `<style>` element.  The selectors may include those used within `@media` rules or those involving dynamic
+     * pseudo-classes (such as `:hover`) or pseudo-elements (such as `::after`).
+     *
+     * @return string[]
+     *
+     * @throws \BadMethodCallException if `inlineCss` has not been called first
+     */
+    public function getMatchingUninlinableSelectors()
+    {
+        if (!isset($this->matchingUninlinableCssRules)) {
```

Comment:
As the field exists, I'd prefer it to (explicitly) be initialized with `null`, and to have an `=== null` check instead of the `isset`. I think that this communicates the intent more clearly.

Reply:
OK